src/components/date.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+
+interface MyDateProps {
```

Comment:
How about we follow the naming pattern set by `GpsCoordStr` and call this component `DateStr`?

Also, maybe name `dateparam` `date` instead.